### PR TITLE
Better monitor

### PR DESCRIPTION
**Pull request by linuxbandit**
Tricky af. still gotta test

### CODE REVIEW

**Comment by serge1peshcoff** [comment]
is it final already?

**Comment by serge1peshcoff** [comment]
@linuxbandit also this: for the future, can you prefix your commits with either `fix(something)` for bug fixes, `feat(something)` for new features` and `chore(something)` for everything else to fully match the conventional commits schema?
and also specify the commit scope if possible, to understand which part of code was touched in the future

File: oms-monitor/docker/docker-compose.yml [hunk 1]
```diff
@@ -6,12 +6,19 @@ services:
...
-      - ./${PATH_OMS_MONITOR}/gf-dashboards:/var/lib/grafana/dashboards
       - ./${PATH_OMS_MONITOR}/config/gf-provisioning:/etc/grafana/provisioning
+      - ./${PATH_OMS_MONITOR}/gf-dashboards/metrics:/var/lib/grafana/dashboards/metrics
+      - ./${PATH_OMS_MONITOR}/gf-dashboards/statistics:/var/lib/grafana/dashboards/statistics
     environment:
       GF_SECURITY_ADMIN_PASSWORD: "${PW_GRAFANA}"
       GF_SERVER_ROOT_URL: "https://${SUBDOMAIN_GRAFANA}${BASE_URL}/"
       GF_INSTALL_PLUGINS: "raintank-worldping-app,grafana-simple-json-datasource,grafana-clock-panel,grafana-piechart-panel"
+      GF_SMTP_ENABLED: "true"
+      GF_SMTP_HOST: "smtp.sendgrid.net:587"
```
Comment (serge1peshcoff) [inline oms-monitor/docker/docker-compose.yml:17]: can we use gsuite for mailing here? I'd go for it instead if possible

Comment (linuxbandit) [reply oms-monitor/docker/docker-compose.yml:17]: not sure I agree, why do you propose that?

Comment (serge1peshcoff) [reply oms-monitor/docker/docker-compose.yml:17]: can't recall lol. this can also work

**Comment by linuxbandit** [comment]
for conventional commit, the scope is optional, so i avoid it when it's not really clear

**Comment by serge1peshcoff** [comment]
yep, it's optional, but better to have it if possible. like `feat(monitoring): something` for monitoring things

**Comment by serge1peshcoff** [comment]
@linuxbandit a suggestion: can you rebase + force push this branch? that way you'll have lint-dockerfile passing (if something else breaking it wasn't added).
let's move to always green :D

**Comment by linuxbandit** [comment]
i can merge master in here, or can close this pr then rebase and open a new one... but won't do dirty things :p

**Review by serge1peshcoff — APPROVED** [review]
too lazy to review, I trust you on that

**Pull request merged**